_posts/2023-02-28-issue-1.md

```diff
@@ -0,0 +1,73 @@
+
+In closing out her speech to the fired up community of supporters and workers, Juana Laura, another fired janitor, reminds us: 
+
+> “Sabemos los derechos son los que nosotros contamos tenemos un unión que nos respalda políticos los medios que gracias a dios ellos están encargados de que todo Estados Unidos o el mundo entero sepan la manera en que estamos siendo desalojados de nuestros trabajo. Vamos a estar allá día y noche hasta que sean respetadas nuestras peticiones. En final, nuestra historia lo vamos a escribir nosotros, no ellos.”
+> 
+> “We know the rights that we can count on. We have the media that is backing us up politically. Thank god the media have made themselves responsible for letting the US and the rest of the world know that we are being let go from our jobs. We’ll fight every day and keep going day and night until they meet our demands. In the end, we are going to be the ones who are going to write our history, not them.”
+
+<hr>
+
+_Thank you to TWC for amplifying this story, and for spreading the word to current and former Twitter coworkers and all people in software. Thank you especially to [Raksha Muthukumar](https://raksha.gay/) for showing up in NYC, talking with us, and compiling this story._
+
+## Announcements
+
+### Research the Life & Legacy of Robert Benner
+
+Marceline Donaldson, a long time women's right and racial justice activist and [former IBM worker we featued last year](https://news.techworkerscoalition.org/2021/07/06/issue-15/), is starting a new research project and looking for people interested in stewarding the effort together. The project is looking into Robert Bennett, the first Black PhD student at Harvard University's Department of Near Eastern Studies, and in the Cambridge area. It is and was an area fraught with extreme racism because those who did such studies were called on for consulting, etc. by the Intelligence Communities. There would not be another Black student in the department for another 20 years. At this stage, Marceline is looking for partners to conceptualize the project. It could take the form of a paper, a documentary, and/or a book. The project is a fascinating opportunity to look into the history of racism in Cambridge and in higher education. Please reach out to Marceline at [bettina-network@comcast.net](bettina-network@comcast.net) if you'd like to get involved!
```

Comment:
```suggestion
Marceline Donaldson, a long time women's right and racial justice activist and [former IBM worker we featued last year](https://news.techworkerscoalition.org/2021/07/06/issue-15/), is starting a new research project and looking for people interested in stewarding the effort together. The project is looking into Robert Bennett, the first Black PhD student at Harvard University's Department of Near Eastern Studies, and in the Cambridge area. It is and was an area fraught with extreme racism because those who did such studies were called on for consulting, etc. by the Intelligence Communities. There would not be another Black student in the department for another 20 years. At this stage, Marceline is looking for partners to conceptualize the project. It could take the form of a paper, a documentary, and/or a book. The project is a fascinating opportunity to look into the history of racism in Cambridge and in higher education. Please reach out to Marceline at [bettina-network@comcast.net](mailto:bettina-network@comcast.net) if you'd like to get involved!
```